apollo-ios-codegen/Sources/GraphQLCompiler/GraphQLSchema.swift

```diff
@@ -59,6 +59,10 @@ public class GraphQLNamedType:
     return lhs.name == rhs.name
   }
 
+  public static func < (lhs: GraphQLNamedType, rhs: GraphQLNamedType) -> Bool {
```

Comment:
I don't think adding `Comparable` to this type is appropriate. I'd rather we just a sort function at the call site in the template.